projects/commudle-admin/src/app/feature-modules/community-channels/components/channel-members/member/member.component.ts

```diff
@@ -31,23 +31,21 @@ export class MemberComponent implements OnInit, OnDestroy {
     private authWatchService: LibAuthwatchService,
     private menuService: NbMenuService,
     private windowService: NbWindowService,
+    private userChatsService: UserChatsService,
     @Inject(NB_WINDOW) private window
-  ) { }
+  ) {
+  }
 
   ngOnInit(): void {
     this.subscriptions.push(
       this.authWatchService.currentUser$.subscribe(
         data => {
           this.currentUser = data;
-          if (data) {
-            this.showLiveStatus = true;
-          } else {
-            this.showLiveStatus = false;
-          }
+          this.showLiveStatus = !!data;
```

Comment:
This is really cool, somehow I forgot it over time. Thanks!

Reply:
My pleasure :)